google/cloud/storage/batch.py

```diff
@@ -219,24 +228,34 @@ def _prepare_batch_request(self):
         _, body = payload.split("\n\n", 1)
         return dict(multi._headers), body, timeout
 
-    def _finish_futures(self, responses):
+    def _finish_futures(self, responses, raise_exception):
```

Comment:
This should default to true according to the docstring; it's important that it defaults to true so that anything that is already using it won't break when we push out the change.

Reply:
updated and added tests